main.c

```diff
@@ -14,8 +14,13 @@
   #include <SDL/SDL_mixer.h>
 #else
   #include "SDL_ttf.h"
+	#if defined(NXDK)
+  #define DISABLE_AUDIO 1
+	#else
+  #define DISABLE_AUDIO 0
```

Comment:
This will enable audio, because you check `#if !defined(DISABLE_AUDIO)`.
That check only checks if `DISABLE_AUDIO` is defined; not which value it has.

I'd avoid defining anything in the case where the platform should have audio. This also shows that it's a very unique situation to disable audio at all.